draft-tuexen-tsvwg-rfc4960-errata.xml

```diff
@@ -733,6 +733,74 @@ partial_bytes_acked and cwnd in the congestion avoidance phase.</t>
 </section>
 </section>
 
+<section title='HEARTBEAT ACK and the association error counter'>
+<section title='Description of the Problem'>
+<t>Section 8.1 and Section 8.3 of <xref target="RFC4960"/>
+prescribe that the receiver of a HEARTBEAT ACK must reset the
+association overall error counter. In some circumstances, e.g. when
+a router discards DATA chunks but not HEARTBEAT chunks due to the
+larger size of the DATA chunk, it is better to not clear the
+association error counter by the HEARTBEAT ACK and reset it by the
+SACK only.</t>
+</section>
```

Comment:
Randy,
I guess the intention is not to have something like the unlucky chunk counter FreeBSD has. So something is needed if a particular DATA chunk is dropped deterministically whenever it is on the wire. I have seen cases where this happens, but the HB procedures keeps the error counters below the thresholds, so the unlucky chunk counter finally clears up things. An alternate protection method could be not to clear the association level error counter on reception of HB-ACKs. My understanding is to modify the specification to allow other methods...

How to deal with PMTUD and PAD chunks is a different story, I think. Irene and myself will write an ID regarding it, since we have an implementation of it...